solr/core/src/test/org/apache/solr/search/TestFilteredDocIdSet.java

```diff
@@ -113,7 +116,7 @@ protected boolean match(int docid) {
   }
   
   public void testNullDocIdSet() throws Exception {
-    // Tests that if a Filter produces a null DocIdSet, which is given to
+    // Tests that if a Query produces a null DocIdSet, which is given to
```

Comment:
Filters produces a DocIdSet; Query does not.  You could try prefixing the comment with "(historical note)" or something

Reply:
Thanks. Prefixed it with :(historical note)" per your suggestion